scene/gui/item_list.cpp

```diff
@@ -35,6 +35,8 @@
 #include "core/string/translation.h"
 #include "scene/theme/theme_db.h"
 
+PropertyListHelper ItemList::base_property_helper;
```

Comment:
I wonder if this is not a bit unnecessary. Considering that you have to implement `_set`, `_get` and `_get_property_list` anyway, it's clear that this it going to be dynamic. Having to create a static object, then create another object out of it is a bit complex IMO.

Maybe we should move the initialization to the constructor? Unless it hurts performance that seems like an acceptable solution.

Another solution would be to, like, find a way to define this object in _bind_methods? So that those stay close together?

Reply:
The old version (https://github.com/godotengine/godot/commit/53ad7ea7151aa58fbd876c8a2ab5fe0030ab23ac#diff-8cbdaea96117950192d3baac12f8cff87b2c4598e04e5df1aae4758f5de3b09b) was being initialized in the constructor, I changed it to static approach. It's definitely cheaper this way and it's closer to how we define the properties normally.